COMMON_ISSUES.md

```diff
@@ -44,6 +44,14 @@ to ensure playback:
   video.muted = true;
   ```
 
+Chrome 76+ Participants downgrade outgoing video bitrate for high audio bitrate limits
+--------------------------------------------------------------------------------------
```

Comment:
According to the bug, this issue is only when REMB is present in the SDP. So this happens only in Group rooms and not for P2P. So should we mention it?